python/py-pyftdi/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-pyftdi
+
+github.setup        eblot pyftdi 0.55.0 v
+github.tarball_from archive
+
+revision            0
+categories          python
+maintainers         nomaintainer
+platforms           any
```

Comment:
as above

Reply:
As in https://github.com/macports/macports-ports/pull/20639#discussion_r1341875995 I don’t see a .so for this Python module.

Reply:
it's about the framework path, so please change to `darwin any`